README.md

```diff
@@ -40,6 +40,10 @@ AWARD_CHANNEL_ID=
 NO_REPORTS_ROLE_ID=
 AWARD_ROLE_ID=
 SILENCED_ROLE_ID=
+GIVEAWAY_DEFAULT_DURATION=
+GIVEAWAY_DEFAULT_WINNERS=
+GIVEAWAY_DEFAULT_PRIZE=
+GIVEAWAY_REACTION_EMOJI
```

Comment:
Shouldn't this line end with =?

Reply:
Yeah, I did not focus enough to the copy pasting job :D